administrator/components/com_languages/src/Controller/InstalledController.php

```diff
@@ -92,8 +92,8 @@ public function switchAdminLanguage()
 			$file = JPATH_ADMINISTRATOR . '/language/' . $cid . '/' . $cid . '.xml';
 		}
 
-		$info         = Installer::parseXMLInstallFile($file);
-		$languageName = $info['name'];
+		$info = LanguageHelper::parseXMLLanguageFile($file);
+		$languageName = $info['nativeName'];
```

Comment:
```suggestion
		$info         = LanguageHelper::parseXMLLanguageFile($file);
		$languageName = $info['nativeName'] ?? $info['name'];
```
1. Restore the alignment of "=" like it was before.
2. To me it seems that we can't rely on the nativeName always being defined, see e.g. this code here: https://github.com/joomla/joomla-cms/blob/4.0-dev/libraries/src/Language/LanguageHelper.php#L52

Reply:
Hmm I'm just told that the nativeName is a required field in XML of J4 languages, so this PR might be ok as it is (except maybe the small code style change).